cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -257,9 +257,18 @@ func (o *Options) config(kubeAPIServerConfig *rest.Config, kubeClient *kubernete
 		}, nil
 	}
 
+	gardenerKubeClient, err := kubernetes.NewForConfig(gardenerAPIServerConfig.ClientConfig)
```

Comment:
Why "gardenerKubeClient"? Isn't it just a "kubernetesClient", not specific to Gardener resources at all?

Reply:
Here client is created from `gardenerAPIServerConfig.ClientConfig` that's why I named it `gardenerKubeClient`.